mne/source_space.py

```diff
@@ -1132,7 +1132,52 @@ def vertex_to_mni(vertices, hemis, subject, subjects_dir=None, mode=None,
     # take point locations in MRI space and convert to MNI coordinates
     xfm = _read_talxfm(subject, subjects_dir, mode)
     data = np.array([rr[h][v, :] for h, v in zip(hemis, vertices)])
-    return apply_trans(xfm['trans'], data)
+    return apply_trans(xfm['trans'], data), xfm
+
+
+##############################################################################
+# Volume to MNI conversion
+
+
+@verbose
+def aseg_vertex_to_mni(vertices, subject, subjects_dir=None, mode=None,
+                       verbose=None):
+    """Convert the array of vertices for a hemisphere to MNI coordinates.
+    Parameters
+    ----------
+    vertices : int, or list of int
+        Vertex number(s) to convert
+    subject : string
+        Name of the subject to load surfaces from.
+    subjects_dir : string, or None
+        Path to SUBJECTS_DIR if it is not set in the environment.
+    mode : string | None
+        Either 'nibabel' or 'freesurfer' for the software to use to
+        obtain the transforms. If None, 'nibabel' is tried first, falling
+        back to 'freesurfer' if it fails. Results should be equivalent with
+        either option, but nibabel may be quicker (and more pythonic).
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see :func:`mne.verbose`
+        and :ref:`Logging documentation <tut_logging>` for more).
+    Returns
+    -------
+    coordinates : n_vertices x 3 array of float
+        The MNI coordinates (in mm) of the vertices
+    Notes
+    -----
+    This function requires either nibabel (in Python) or Freesurfer
+    (with utility "mri_info") to be correctly installed.
+    """
+    if not has_freesurfer() and not has_nibabel():
+        raise RuntimeError('NiBabel (Python) or Freesurfer (Unix) must be '
+                           'correctly installed and accessible from Python')
+
+    if not isinstance(vertices, list) and not isinstance(vertices, np.ndarray):
+        vertices = [vertices]
+
+    # take point locations in MRI space and convert to MNI coordinates
+    xfm = _read_talxfm(subject, subjects_dir, mode)
+    return apply_trans(xfm['trans'], vertices), xfm
```

Comment:
@agramfort I'll check the reason travis is not happy! Thks!

@Eric89GXL ok, I'll use `_read_talxfm`. Thks!